federatedscope/caesar_v_fl/Paillier/abstract_paillier.py

```diff
@@ -0,0 +1,47 @@
+# You can refer to pyphe for the detail implementation. (
```

Comment:
This file is the same as `federatedscope/vertical_fl/Paillier/abstract_paillier.py`, maybe you can reuse it.